robot_smach_states/src/robot_smach_states/navigation/follow_operator.py

```diff
@@ -88,7 +88,7 @@ def _operator_standing_still_for_x_seconds(self, timeout):
         return False
 
     def _standing_still_for_x_seconds(self, timeout):
-        current_pose_stamped = self._robot.base.get_location()
+        current_pose_stamped = self._robot.base.get_location().frame
```

Comment:
This is called stamped, while you actually remove the stamp. Although it was 'wrong' before, I'd change it